certbot/error_handler.py

```diff
@@ -70,8 +70,12 @@ def __exit__(self, exec_type, exec_value, trace):
         self.body_executed = True
         retval = False
         # SystemExit is ignored to properly handle forks that don't exec
-        if exec_type in (None, SystemExit):
+        if exec_type is SystemExit:
             return retval
+        elif exec_type is None:
+            if not self.call_on_regular_exit:
+                return retval
+            retval = True
```

Comment:
I'm not sure it matters, but if an exception wasn't raised, why do we want to we want to return `True` if we call the registered functions and `False` if we don't? If it doesn't matter, I'd personally leave `retval` alone and not set it here.